src/contexts/ConfigurationContext.tsx

```diff
@@ -13,7 +13,7 @@ import {
   UxMode,
   normal_reranker_id,
   mmr_reranker_id,
-  FcsMode, FCS_MODE,
+  FcsMode, FCS_MODE, slingshot_reranker_id
```

Comment:
Why do you need the slingshot_reranker_id as a config param? I thought it could be just in the code, no?

Reply:
its in code, just importing in the config to set the reranker id for https://github.com/vectara/vectara-answer/pull/155/files#diff-3a445224bc341427785ac65b7124070512bb1920fdd07cb979a4deb9d7f87bf4R157